### ISSUE TITLE
I compress a video on Android but i can't play it on browser just audio

### ISSUE THREAD

**Issue by gasparnd** [numandev1/react-native-compressor#163]
### Current behavior
I have installed the library in a expo ejected project and successfully configured it on Android environment. The video compression works well but the compressed video don't play in browser just the sound. The problem is in Chrome, in Safari i'm able to see it. I'm able to play it well if i download the video

### Expected behavior
Compress a video in Android and IOS and play it on the browser, both picture and sound

### Platform

- [✅] Android
- [❌] iOS

### Environment Version
```
node: v18.16.0
OS: macOS ventura v13.4.1
```

### React Native Version
```
"expo": "~48.0.18",
"expo-image-picker": "~14.1.1",
"react-native": "0.71.8"
```

### React Native Compressor Version
```
"react-native-compressor": "^1.6.3",
```

### Reproducible Steps And Demo
Create an Expo app, then generate a prebuilt. Last step install and set up the libraries expo-image-picker and react-native-compressor

Picker and compression code (there is more code but this is the more important one):

```
const onCompressVideo = async (videoUri: string) => {
  const result = await Video.compress(
    videoUri,
    {
      compressionMethod: "auto",
    },
    (progress) => {
      console.log(
        "🚀 ~ file: index.tsx:52 ~ onCompressVideo ~ progress:",
        progress
      );
    }
  );
  console.log("🚀 ~ file: index.tsx:56 ~ onCompressVideo ~ result:", result);

  return result;
};

let res = await ImagePicker.launchImageLibraryAsync({
  mediaTypes: ImagePicker.MediaTypeOptions.Videos,
  videoQuality:
   ImagePicker.UIImagePickerControllerQualityType.IFrame960x540,
});

const compressedVideo = await onCompressVideo(res.assets[0].uri);

onUploadVideo(compressedVideo)


```

**Comment by paulsizer** [comment]
I am also seeing this problem, It appears to be in the newer chrome update (v115) so not sure what is happening here.

**Comment by gasparnd** [comment]
@paulsizer I was able to do it works in Chrome. Basically one must go to **settings > system > and disable the "Use hardware acceleration when available" switch**. This action made the video works on the browser, now i can play it normally and download. My question now is why a compressed video doesn't works without this and a normal video do? And how i can get it works?

**Comment by paulsizer** [comment]
@gasparnd Exactly the fix we have put out to clients until we get to the bottom of this.

**Comment by gasparnd** [comment]
> @gasparnd Exactly the fix we have put out to clients until we get to the bottom of this.

For sure, we too. It's sad to do that have an impact in the UX. I need to look around for a solution. Anyway it seems like a Browser problem. Please keep in touch if found a solution.

**Comment by sauravjmedhi-softsensor** [comment]
Facing the same issue. After compression, the video does not play in Chrome browser (only audio), but Firefox is able to play the video. I tried disabling **"Use hardware acceleration when available"**, but that didn't help.
This issue started popping up today out of the blue.

I have noticed one difference between the video that was playable in Chrome, vs, the one that wasn't. Upon checking the video codecs, **MOOV Atom Location** was at **middle** position for the bad video, but it was at **beginning** position for the good video where it should be.

Can anyone check the video codecs and could this be the issue?

**Comment by sauravjmedhi-softsensor** [comment]
> > @gasparnd Exactly the fix we have put out to clients until we get to the bottom of this.
> 
> For sure, we too. It's sad to do that have an impact in the UX. I need to look around for a solution. Anyway it seems like a Browser problem. Please keep in touch if found a solution.

Did you find any solution? This issue came up two days ago, and haven't been able to find any fix yet. Only the **hardware acceleration** thing works.

**Comment by gasparnd** [comment]
I'm so sorry @sauravjmedhi-softsensor. I don't found other solution for this. :(

**Comment by numandev1** [comment]
🙏 sorry for a long ago I was not maintaining due to the huge office workload, hopefully, I am going to start fixing bugs on a weekend basis,
I will fix at least 2 issues on the weekend, for priority bugs you can join the discord channel https://discord.gg/dtYzk8sp where we can discuss which bug is most important

**Comment by paulsizer** [comment]
This looks to be fixed in Chrome v116, I am now seeing the video with sound

**Comment by numandev1** [comment]
@gasparnd @sauravjmedhi-softsensor can you confirm if this issue has gone?

**Comment by numandev1** [comment]
fixed by https://github.com/numandev1/react-native-compressor/pull/180

**Comment by numandev1** [comment]
fixed in `1.8.2`

**Comment by gasparnd** [comment]
Sorry for the time. Iy was fixed
